proposals/005-namespacing-govuk-frontend.md

```diff
@@ -0,0 +1,132 @@
+# Namespacing GOV.UK Frontend
```

Comment:
Need to make this title more specific since we'll have another namespacing proposal.